src/main/java/org/jooq/lambda/Unchecked.java

```diff
@@ -46,6 +46,9 @@
 
         if (t instanceof RuntimeException)
             throw (RuntimeException) t;
+
+        if (t instanceof IOException)
+            throw new UncheckedIOException((IOException) t);
```

Comment:
Of course, my bad. IntelliJ changed `import java.util.function.*` to bunch of imports, so I ignored them when doing `git add -p` ;)